README.md

```diff
@@ -53,12 +53,30 @@ class { 'puppetwebhook':
 
 ## Limitations
 
-Currently does not support the following:
+The rpm/deb packages expect that you provide a running redis instance.
```

Comment:
or a remote redis connection

Reply:
that's not yet supported by puppet-webhook as far as I know. 127.0.0.1 is hardcoded as redis endpoint.